pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/InMemorySendingMailbox.java

```diff
@@ -106,4 +106,9 @@ public boolean isEarlyTerminated() {
   public boolean isTerminated() {
     return _isTerminated;
   }
+
+  @Override
+  public String toString() {
+    return "m" + _id;
```

Comment:
`g` and `m` might not be very easily understandable prefixes; could we use `grpc |` / `memory |` as prefixes to the mailbox ID instead?

Reply:
I had to balance between long and easier to understand ids and short ones. I would usually prefer the longer, but the logs that use these ids are already very long, so I think having longer ids makes them even more difficult to read. Also, given there are only 3 types of mailboxes and each one is named in a way that the first character is unique, I think it is not very difficult to read.